nixos/modules/services/cluster/kubernetes/kubelet.nix

```
@@ -336,7 +336,10 @@ in
           [
             gitMinimal
             openssh
-            util-linux
+            # TODO (#409339): remove this patch. We had to add it to avoid a mass rebuild
```

Not sure if I'm missing something, but I think the staging cycle for https://github.com/NixOS/nixpkgs/pull/405952 has to complete to switch back.

```suggestion
            # TODO (#405952): remove this patch. We had to add it to avoid a mass rebuild
```

---

I just made that tracking issue to keep track of all the PRs where we changed something and some cleanup will be required (including the staging cycle): #409339

---

Thx, makes sense

---

I got it